From 7b0ef55edd63c71be209aa3b57e2190d4ce4fb90 Mon Sep 17 00:00:00 2001
From: Mart Raudsepp <leio@gentoo.org>
Date: Sun, 3 Oct 2021 23:52:27 +0300
Subject: [PATCH] build: Don't automagically depend on webkit2gtk-4.1

That's the libsoup3 using webkit-gtk SLOT that we don't have. Patch it out,
so it wouldn't be picked up automagically once it's added or comes from some
overlay.

This reverts commit f9d52aad23914df9dcb404723d1c99cf7b4e1eca.
---
 meson.build | 7 +------
 1 file changed, 1 insertion(+), 6 deletions(-)

diff --git a/meson.build b/meson.build
index e57e466e..492aafed 100644
--- a/meson.build
+++ b/meson.build
@@ -46,15 +46,10 @@ lt_revision = 3
 lt_age = 0
 LIBDEVHELP_LT_VERSION = '@0@.@1@.@2@'.format(lt_current, lt_revision, lt_age)
 
-webkit_dep = dependency('webkit2gtk-4.1', required : false)
-if not webkit_dep.found()
-  webkit_dep = dependency('webkit2gtk-4.0')
-endif
-
 LIBDEVHELP_PUBLIC_DEPS = [
   dependency('gio-2.0', version: '>= 2.64'),
   dependency('gtk+-3.0', version: '>= 3.22'),
-  webkit_dep,
+  dependency('webkit2gtk-4.0', version: '>= 2.26'),
 ]
 LIBDEVHELP_PRIVATE_DEPS = [
   dependency('gsettings-desktop-schemas'),
-- 
2.32.0

